assets/dev/js/editor/elements/views/base.js

```diff
@@ -85,18 +90,13 @@ BaseElementView = BaseContainer.extend( {
 	},
 
 	getChildView( model ) {
-		let ChildView;
-		const elType = model.get( 'elType' );
+		const elementType = elementor.getElementType( model.get( 'elType' ), model.get( 'widgetType' ) );
```

Comment:
Does it return a type name or an actual class instance?
```suggestion
		const elementClass = elementor.getElementType( model.get( 'elType' ), model.get( 'widgetType' ) );
```

Reply:
it is instance.